tests/time-period-checkout.test.js

```
@@ -199,4 +208,45 @@ describe("time period checkout and availability rules", () => {
+      preparationLeadTimeMinutes: 120,
+      serviceHours: SERVICE_HOURS,
+    });
+    const mondaySlot = generateTimePeriodInstances(
+      localDate("2026-06-15"),
+      localDate("2026-06-16"),
+      [morningSlot],
+    ).find((slot) => new Date(slot.timeBegin).getDay() === 1);
+    const provider = buildProvider(bookable);
+    const originalMembershipLookup =
+      MembershipManager.getMembershipsByTenantAndRoles;
+    const clock = sinon.useFakeTimers(localDate("2026-06-15", "09:30").getTime());
```

ESLint

[error] Replace `localDate("2026-06-15",·"09:30").getTime()` with `⏎······localDate("2026-06-15",·"09:30").getTime(),⏎····`

(prettier/prettier)